metrics/cpu_enabled.go

```diff
@@ -14,7 +14,7 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
 
-// +build !ios
+// +build darwin,cgo dragonfly freebsd linux nacl netbsd openbsd solaris windows arm64
```

Comment:
This looks very weird. Architectures mixed with platforms. It would also be a maintenance nightmare to keep it always updated. Please use exclusion patterns

Reply:
Ping @BogdanHabic